docs/source/guides/network_node/staking_guide.rst

```diff
@@ -113,11 +122,11 @@ Running an Ethereum Node for Staking (Local Provider)
 Here we describe the steps required to run an ethereum node for both transaction signing and broadcast.
 This is the typical configuration for a locally operated trusted ethereum node.
 
-Assuming you have ``geth`` installed, let's run a node on the Görli testnet.
+Assuming you have ``geth`` installed
```

Comment:
This guide has fallen in relevance with the induction of hosted node solutions.  Perhaps a follow up Issue is needed to either modify this guide, or make another one for non-local ethereum providers.

Reply:
✔️ - Filed #1952